decks/node-beginner/meeting-1.mdx

```diff
@@ -0,0 +1,408 @@
+import { MinuteTimer } from '../components/minute-timer';
+import { Split } from '@mdx-deck/layouts';
+export { theme } from './theme.js';
+import { Link } from '@reach/router';
+import CodeLayout from './code-layout';
+
+# [Digital Products Boosters](https://accenture.github.io/digital-products-boosters)
+
+## Node Beginner - Meeting 1
+
+<Split>
+  <div>
+    <div style={{ 'text-align': 'left' }}>Meeting Pre-work</div>
+    <ol>
+      <li>
+        <Link to="/1">Reading 📕</Link>
+      </li>
+      <li>
+        <Link to="/2">Setup 🛠</Link>
+      </li>
+    </ol>
+  </div>
+  <div>
+    <div style={{ 'text-align': 'left' }}>Meeting Agenda</div>
+    <ol>
+      <li>
+        <Link to="/3">Discussion Questions 🤔</Link>
+      </li>
+      <li>
+        <Link to="/10">Exercises 💪🏼</Link>
+      </li>
+    </ol>
+  </div>
+</Split>
+
+---
+
+## Reading 📕
+
+1. [Core ES6 - From var to const/let](https://exploringjs.com/es6/ch_core-features.html#sec_from-var-to-const)
+2. [Core ES6 - Scoping](https://exploringjs.com/es6/ch_variables.html)
+3. [Core ES6 - this Keyword](https://www.codementor.io/@dariogarciamoya/understanding--this--in-javascript-du1084lyn)
+4. [Core ES6 - this Keyword with Arrow Function](https://www.codementor.io/@dariogarciamoya/understanding-this-in-javascript-with-arrow-functions-gcpjwfyuc)
+5. [Core ES6 - Promises, Async, Await](https://javascript.info/async)
+6. [Core ES6 - Template Literals](https://wesbos.com/tagged-template-literals/)
+7. [Node Design Patterns - The Module System](http://ahmed-badawy.com/blog/wp-content/uploads/2018/10/Node.js-Design-Patterns-2nd-Edition-by-Mario-Casciaro-Luciano-Mammino.pdf), page 47 - 61
+
+---
+
+<CodeLayout>
+
+## Setup 🛠
+
+1. Do the Boosters [Initial Setup](https://github.com/Accenture/digital-products-boosters#initial-setup) if you have not done so.
+
+2. From the root of the `digital-products-boosters` repository, run:
+
+```bash
+$ cd boosters/react-beginner/meeting-1
```

Comment:
Directory path should change to `boosters/node-beginner/meeting-1` if there is a project there to use for the exercises. If not, we can remove this setup page, or re-word it accordingly.